### PR TITLE
Make error message configurable by HTTP call

### PR DESCRIPTION
**Pull request by nathanleclaire**
Signed-off-by: Nathan LeClaire nathan.leclaire@gmail.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Deprecated
 
-This project is officially deprecated in favor of [Docker Machine](https://docs.docker.com/machine/).  The code and documentation here only exist as a reference for users who have not yet switched over (but please do soon).
+This project is officially deprecated in favor of [Docker
+Machine](https://docs.docker.com/machine/).  The code and documentation here
+only exist as a reference for users who have not yet switched over (but please
+do soon).
```
Comment (tianon) [inline README.md]: Intentional or unrelated?

Comment (tianon) [inline README.md]: Probably should also mention Docker Toolbox like the updated fallback message does.

**Comment by tianon** [comment]
I really hate pulling this over the network, but OK, I give in. :smile:

File: README.md (hunk 1 above)
Comment (nathanleclaire) [inline README.md]: Unrelated word wrap pedantism, it was in the original PR so I updated it here.

Comment (nathanleclaire) [inline README.md]: > Probably should also mention Docker Toolbox like the updated fallback message does.

Yeah, good idea.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Deprecated
 
-This project is officially deprecated in favor of [Docker Machine](https://docs.docker.com/machine/).  The code and documentation here only exist as a reference for users who have not yet switched over (but please do soon).
+This project is officially deprecated in favor of [Docker
+Machine](https://docs.docker.com/machine/).  The code and documentation here
+only exist as a reference for users who have not yet switched over (but please
+do soon).  The recommended way to install Machine is with the [Docker
+Toolbox](https://docker.com/toolbox).
```
Comment (tianon) [inline README.md:7]: Lol, is https://docker.com/toolbox supposed to be a 404?

Comment (nathanleclaire) [inline README.md:7]: It's not up yet, but @mchiang @jeffdm tell me that will be the final URL.

**Pull request merged**